src/models/basecog.py

```diff
@@ -7,16 +8,16 @@ def __init__(self, bot):
 
     async def cog_check(self, ctx):
         return (
-            ctx.channel.id == 960446827579199488
-            or 830875873027817484 in [role.id for role in ctx.author.roles]
-            or 959723229805707285 in [role.id for role in ctx.author.roles]
+            ctx.channel.id == config.suggestion_channel
+            or config.role1 in [role.id for role in ctx.author.roles]
+            or config.role2 in [role.id for role in ctx.author.roles]
             or self.bot.is_owner(ctx.author)
         )
 
     def cog_application_command_check(self, interaction):
         return (
-            interaction.channel.id == 960446827579199488
-            or 830875873027817484 in [role.id for role in interaction.user.roles]
-            or 959723229805707285 in [role.id for role in interaction.user.roles]
+            interaction.channel.id == int(config.suggestion_channel)
```

Comment:
why should config.suggestion_channel converted to int?

Reply:
because that is how d.py wanted it

Reply:
this wasnt developed with dpy, but nextcord :(

Reply:
but iirc, still decent and acceptable.